packages/flex-plugin-webpack/src/compiler.ts

```diff
@@ -74,20 +74,19 @@ export default (
         });
       });
 
-      ForkTsCheckerWebpackPlugin.getCompilerHooks(compiler).receive.tap(
-        'afterTSCheck',
-        (diagnostics: Issue[], lints: Issue[]) => {
-          const allMsgs = [...diagnostics, ...lints];
-          const format = (issue: Issue) => `${issue.file}\n${typescriptFormatter(issue)}`;
-
-          if (tsMessagesResolver) {
-            tsMessagesResolver({
-              errors: allMsgs.filter((msg) => msg.severity === 'error').map(format),
-              warnings: allMsgs.filter((msg) => msg.severity === 'warning').map(format),
-            });
-          }
-        },
-      );
+      // Not sure if this is correct bc the recieve hook no longer exists and now hooks.tap require a return.
```

Comment:
For reference [here](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/main/src/plugin-hooks.ts) is how the new hooks work with this webpack plugin.